doc/main/tbb_userguide/Migration_Guide/Task_API.rst

```diff
@@ -325,3 +325,47 @@ task:
         }
         tg.wait();
     }
+
+
+Scheduler Bypass
+----------------
+
+In TBB task::execute() method can return a pointer to task to be executed next. 
+Thus both avoids spawn overhead and prevents steal of the task by other thread.
+
+.. code:: cpp
+    
+    #include <tbb/task.h>
+    
+    // Assuming OtherTask is defined.
+    
+    struct Task : tbb::task {
+        task* execute(){
+            // some work to do
+            return new(tbb::task::allocate_root()) OtherTask{};
+        }
+    };
+    
+    int main(){
+        tbb::task* r = new (tbb::task::allocate_root()) Task{};
+        task::spawn_root_and_wait(r);
+    }
+
+In oneTBB this can be done using preview feature of ``oneapi::tbb::task_group``. 
```

Comment:
```suggestion
In oneTBB this can be done using the preview feature of ``oneapi::tbb::task_group``. 
```

Reply:
Add some description what `defer` is doing that it creates a task for deferred execution that can be bypassed similar to classic TBB.